status/status.go

```diff
@@ -216,3 +220,18 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+// GenerateURLrefferenceToConsole returns string link to integration test pipelinerun logs for tenant
+func GenerateURLrefferenceToConsole(application string, pipelinerun string, namespace string) string {
+	console_url := os.Getenv("CONSOLE_URL")
+	if console_url == "" {
+		return "ENVVARNOTFOUND"
+	}
+
+	papw := "/preview/application-pipeline/workspaces/"
+	apps := "/applications/"
+	plrs := "/pipelineruns/"
+	logs := "/logs"
```

Comment:
Do we really want to point directly to logs?

Reply:
Removed logs.